x/stakeibc/keeper/deposit_records.go

```diff
@@ -86,46 +94,57 @@ func (k Keeper) TransferExistingDepositsToHostZones(ctx sdk.Context, epochNumber
 			k.Logger(ctx).Error(fmt.Sprintf("\t[TransferExistingDepositsToHostZones] err {%s}", err.Error()))
 			continue
 		}
+
+		k.Logger(ctx).Info(utils.LogWithHostZone(hostZone.ChainId, "Successfully submitted transfer"))
 	}
 }
 
+// Iterate each deposit record marked DELEGATION_QUEUE and use the delegation ICA to delegate on the host zone
 func (k Keeper) StakeExistingDepositsOnHostZones(ctx sdk.Context, epochNumber uint64, depositRecords []recordstypes.DepositRecord) {
+	k.Logger(ctx).Info("Staking deposit records...")
+
 	stakeDepositRecords := utils.FilterDepositRecords(depositRecords, func(record recordstypes.DepositRecord) (condition bool) {
 		isStakeRecord := record.Status == recordstypes.DepositRecord_DELEGATION_QUEUE
 		isBeforeCurrentEpoch := record.DepositEpochNumber < epochNumber
 		return isStakeRecord && isBeforeCurrentEpoch
 	})
 
+	if len(stakeDepositRecords) == 0 {
+		k.Logger(ctx).Info("No deposit records in state DELEGATION_QUEUE")
+		return
+	}
+
 	// limit the number of staking deposits to process per epoch
 	maxDepositRecordsToStake := utils.Min(len(stakeDepositRecords), cast.ToInt(k.GetParam(ctx, types.KeyMaxStakeICACallsPerEpoch)))
-	k.Logger(ctx).Info(fmt.Sprintf("Staking %d out of %d deposit records", maxDepositRecordsToStake, len(stakeDepositRecords)))
+	if maxDepositRecordsToStake < len(stakeDepositRecords) {
+		k.Logger(ctx).Info(fmt.Sprintf("  MaxStakeICACallsPerEpoch limit reached - Only staking %d out of %d deposit records", maxDepositRecordsToStake, len(stakeDepositRecords)))
+	}
 
 	for _, depositRecord := range stakeDepositRecords[:maxDepositRecordsToStake] {
-		k.Logger(ctx).Info(fmt.Sprintf("\t[StakeExistingDepositsOnHostZones] Processing deposit ID:{%d} DENOM:{%s} AMT:{%d}",
-			depositRecord.Id, depositRecord.Denom, depositRecord.Amount))
+		k.Logger(ctx).Info(utils.LogWithHostZone(depositRecord.HostZoneId,
+			"Processing deposit record %d: %d%s", depositRecord.Id, depositRecord.Amount, depositRecord.Denom))
 
 		hostZone, hostZoneFound := k.GetHostZone(ctx, depositRecord.HostZoneId)
 		if !hostZoneFound {
 			k.Logger(ctx).Error(fmt.Sprintf("[StakeExistingDepositsOnHostZones] Host zone not found for deposit record {%d}", depositRecord.Id))
```

Comment:
Are these tags, e.g. `StakeExistingDepositsOnHostZones`, necessary? Not a big deal to keep them in, but seems a bit redundant given the log on line 125

Reply:
That's an error log so it shouldn't be printed in the normal flow!
I purposefully left errors untouched in this PR considering there's quite a bit we could do to clean those up and I don't think we want them to follow the same flow as the normal logs since we want them to jump out